ckcp/openshift_dev_setup.sh

```diff
@@ -10,26 +10,29 @@ SCRIPT_DIR="$(
 
 GITOPS_DIR="$(dirname "$SCRIPT_DIR")/gitops"
 CKCP_DIR="$(dirname "$SCRIPT_DIR")/ckcp"
-WORK_DIR="$SCRIPT_DIR/work"
 KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
-KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/admin.kubeconfig.base"
 CR_TO_SYNC=(
-            conditions.tekton.dev
-            pipelines.tekton.dev
-            pipelineruns.tekton.dev
-            pipelineresources.tekton.dev
-            runs.tekton.dev
-            tasks.tekton.dev
-          )
+  conditions.tekton.dev
+  pipelines.tekton.dev
+  pipelineruns.tekton.dev
+  pipelineresources.tekton.dev
+  runs.tekton.dev
+  tasks.tekton.dev
+)
```

Comment:
This is really a code style preference thing. We should avoid such changes when they are not driven by clear guidelines. As a side note this will produce a merge conflict with PR #148

Reply:
IDE autoformatting did too much, fixed :)